packages/framework-core/test/booster-command-dispatcher.test.ts

```diff
@@ -193,5 +195,77 @@ describe('the `BoosterCommandsDispatcher`', () => {
       })
       expect(asyncOperationFinished).to.be.true
     })
+
+    context('when before hook functions are passed', () => {
+      const newComment = 'Look, I changed the message'
+      const newCommentV2 = 'Yes, I changed it for a second time'
+      const beforeFn: CommandBeforeFunction = (input, currentUser) => {
+        input.comment = newComment
+        return input
+      }
+      const beforeFnV2: CommandBeforeFunction = (input, currentUser) => {
+        // To double-check it's really chained
+        if (input.comment === newComment) input.comment = newCommentV2
+        return input
+      }
+
+      it('transforms the input if a before hook function is passed', async () => {
+        let transformedInput = {}
+        @Command({ authorize: 'all', beforeCommand: [beforeFn] })
+        class PostComment {
+          public constructor(readonly comment: string) {}
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
+          public static async handle(command: PostComment, _register: Register): Promise<void> {
```

Comment:
I believe you can remove the register arg
```suggestion
          public static async handle(command: PostComment): Promise<void> {
```